Hashids/Hashids.swift

```diff
@@ -305,7 +307,7 @@ func transform<T: CollectionType where T.Generator.Element: Equatable>(a: T, _ b
 }
 
 func unique<T: CollectionType where T.Generator.Element: Equatable>(a: T) -> [T.Generator.Element] {
-    return transform(a, a) { (var c, a, b, e) in
+    return transform(a, a) { (inout c: [T.Generator.Element], a, b, e) in
```

Comment:
it was just the `var parameters are deprecated and will be removed in Swift 3` but all I did just now was remove the var and it passed all the tests...so now I have to think about this more because I'm confused